docs/changes/0.14.0.rst

```diff
@@ -19,35 +19,40 @@ _________________
 * Remove deprecated ``zmq_helpers.py`` with ``Publisher`` and ``UnboundedPublisher`` (#1983)
 * Remove deprecated ``_save_val`` method of ``_BaseParameter`` (#1980)
 * Remove deprecated ``qcodes.dataset.sqlite_base`` and ``qcodes.dataset.database`` modules (#1981)
-* The ``name`` and the ``short_name`` attributes of the _BaseParameter have become read-only. It is not possible to change the name or short_name of the parameter after instantiation. (#1989)
-* ..
+* The ``name`` and the ``short_name`` attributes of the _BaseParameter have become read-only. It is not 
+  possible to change the name or short_name of the parameter after instantiation. (#1989)
+
 
 Improved drivers:
 _________________
 
-* ..
+* Improvements to ``KeysightB1520A`` to fix the following two issues. First, ``is_enabled`` function outputs 
+  `False` instead of throwing an error if channels are not enabled. Second, the values of ``voltage_dc``, 
+  ``voltage_ac``, and ``frequency`` parameters can be raised/decreased at a specified rate by defining the 
+  step attribute of these parameters. (#1998)
+* New method to increase or decrease the range of the ``Keysight344xx`` drivers. This method 
+  can access and search ranges and implements them into the setter. (#1996)
+* Coerce ``KeysightB1500`` coefficient for nplc settings to int. The b1500 instrument does not update 
+  integration time if it receives a non-int value. (#1987)
+
```

Comment:
@trevormorgan if #2001 is merged. You may use the following description 
" Added parameters to``KeysightB1520A`` driver to allow CV staircase sweep. The ``setup_staircase_cv`` method can be used to set-up the state of the CMU for CV staircase sweep and ``run_sweep`` method to run the sweep measurements. Look at CV Sweep section of example notebook for more description."